### ISSUE TITLE
tsconfig paths mappings do not seem to be used for style files

### ISSUE THREAD

**Issue by renehamburger** [monounity/karma-typescript#263]
In my repository [blinx.js](https://github.com/renehamburger/blinx.js) I'm using a single path mapping:

```
    "paths": {
      "*": ["./*"]
    },
```

karma-typescript is able to resolve all *.ts and *.js files using a path that is resolved through this mapping, but it was failing to load the css file loaded [here](https://github.com/renehamburger/blinx.js/blob/master/src/blinx.class.ts#L15) with that path alias. I have since changed it to a relative path, but if you move back to `import 'src/css/blinx.css'` you should be able to reproduce the issue.

**Comment by erikbarke** [comment]
Yeah, this isn't supported yet but I'll try to come up with something!
